### PR TITLE
Problem #242: several more test-specific CSS-classes were found

### PR DESCRIPTION
**Pull request by kegor**
Additional changes for #242.

### CODE REVIEW

File: src/KitchenSink/wwwroot/KitchenSink/LazyLoadingPage.html [hunk 1]
```diff
@@ -7,7 +7,7 @@ <h1 slot="kitchensink/lazyLoading-heading" class="kitchensink-heading-1">Lazy Lo
         <p slot="kitchensink/lazyLoading-scheduling-description"> By using <code>Scheduling.RunTask</code> it is possible to achieve asynchronous data loading. </p>
         <p slot="kitchensink/lazyLoading-table-description"> In the table below, one can retrieve data for every person by hovering them. As a person gets hovered, it will automatically start loading that persons data. Once the data has been loaded, it will fill the "Favorite Game" column. </p>
 
-        <table slot="kitchensink/lazyLoading-table" class="table table-striped table-hover table-sortable-list">
+        <table slot="kitchensink/lazyLoading-table" class="table-hover table-sortable-list">
```
Comment (kegor) [inline src/KitchenSink/wwwroot/KitchenSink/LazyLoadingPage.html]: Although I can't find other usages for these classes anywhere, they are not used in the tests, so in my view, they are not in the scope of current PR.

**Comment by warpech** [comment]
As it was agreed on the planning meeting, all PRs should be made to https://github.com/Starcounter/KitchenSink/pull/241 branch until it is merged.

@kegor can you please retarget the PR?

File: src/KitchenSink/wwwroot/KitchenSink/FileUploadPage.html [hunk 2]
```diff
@@ -34,7 +34,7 @@ <h2 slot="kitchensink/fileupload-upload-file-header">Uploaded files</h2>
                     </template>
                     </tbody>
                 </table>
-                <div slot="kitchensink/fileupload-files-table-message" class="kitchensink-test-warning">
+                <div slot = "kitchensink/fileupload-files-table-message">
```
Comment (warpech) [inline src/KitchenSink/wwwroot/KitchenSink/FileUploadPage.html]: There should be no whitespace around `=`.

Comment (kegor) [reply src/KitchenSink/wwwroot/KitchenSink/FileUploadPage.html]: Done.

**Comment by warpech** [comment]
I changed the base branch to `master-2.4`, because https://github.com/Starcounter/KitchenSink/pull/241 was merged to `master-2.4` yesterday.

I will review today.

**Pull request merged**